exec.c

```diff
@@ -0,0 +1,30 @@
+#include <common.h>
+#include <exec.h>
+
+int main() {
+  // This is a simple demonstration of the exec functionality. It
```

Comment:
I will. My comments are pretty verbose, so it's better to write them using a consistent style.